### PR TITLE
authentication

### PR DESCRIPTION
**Pull request by duchk-2510**
image
Login, logout
https://edu-redmine.sun-asterisk.vn/issues/36083

### CODE REVIEW

**Comment by duchk-2510** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "RubyFresherEcommerce"
```
Comment (vanvtt-0952) [inline app/helpers/application_helper.rb]: I18n anh bạn

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -6,14 +6,27 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <%= link_to "Rails eCommerce", "#", class: "navbar-brand" %>
+      <%= link_to "Rails eCommerce", root_path,class: "navbar-brand" %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_navbar.html.erb]: `"Rails eCommerce",`

Comment (vanvtt-0952) [reply app/views/layouts/_navbar.html.erb]: `,class` -> `, class`

**Comment by duchk-2510** [comment]
ready

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Rails eCommerce"
```
Comment (vanvtt-0952) [inline app/helpers/application_helper.rb:3]: `"Rails eCommerce"` I18n

**Pull request merged**